src/Node/FS/Async.purs

```diff
@@ -64,42 +64,41 @@ foreign import handleCallbackImpl ::
-  , unlink :: Fn2 FilePath (JSCallback Unit) Unit
-  , rmdir :: Fn2 FilePath (JSCallback Unit) Unit
-  , mkdir :: Fn3 FilePath { recursive :: Boolean, mode :: String } (JSCallback Unit) Unit
-  , readdir :: Fn2 FilePath (JSCallback (Array FilePath)) Unit
-  , utimes :: Fn4 FilePath Int Int (JSCallback Unit) Unit
-  , readFile :: forall a opts. Fn3 FilePath { | opts } (JSCallback a) Unit
-  , writeFile :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
-  , appendFile :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
-  , exists :: forall a. Fn2 FilePath (Boolean -> a) Unit
-  , open :: Fn4 FilePath String (Nullable FileMode) (JSCallback FileDescriptor) Unit
-  , read :: Fn6 FileDescriptor Buffer BufferOffset BufferLength (Nullable FilePosition) (JSCallback ByteCount) Unit
-  , write :: Fn6 FileDescriptor Buffer BufferOffset BufferLength (Nullable FilePosition) (JSCallback ByteCount) Unit
-  , close :: Fn2 FileDescriptor (JSCallback Unit) Unit
-  }
-fs = unsafeRequireFS
 
 -- | Type synonym for callback functions.
 type Callback a = Either Error a -> Effect Unit
 
+foreign import rename_ :: Fn3 FilePath FilePath (JSCallback Unit) Unit
+foreign import truncate_ :: Fn3 FilePath Int (JSCallback Unit) Unit
+foreign import chown_ :: Fn4 FilePath Int Int (JSCallback Unit) Unit
+foreign import chmod_ :: Fn3 FilePath String (JSCallback Unit) Unit
+foreign import stat_ :: Fn2 FilePath (JSCallback StatsObj) Unit
+foreign import link_ :: Fn3 FilePath FilePath (JSCallback Unit) Unit
+foreign import symlink_ :: Fn4 FilePath FilePath String (JSCallback Unit) Unit
+foreign import readlink_ :: Fn2 FilePath (JSCallback FilePath) Unit
+foreign import realpath_ :: forall cache. Fn3 FilePath { | cache } (JSCallback FilePath) Unit
+foreign import unlink_ :: Fn2 FilePath (JSCallback Unit) Unit
+foreign import rmdir_ :: Fn2 FilePath (JSCallback Unit) Unit
+foreign import mkdir_ :: Fn3 FilePath { recursive :: Boolean, mode :: String } (JSCallback Unit) Unit
+foreign import readdir_ :: Fn2 FilePath (JSCallback (Array FilePath)) Unit
+foreign import utimes_ :: Fn4 FilePath Int Int (JSCallback Unit) Unit
+foreign import readFile_ :: forall a opts. Fn3 FilePath { | opts } (JSCallback a) Unit
+foreign import writeFile_ :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
+foreign import appendFile_ :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
+foreign import exists_ :: forall a. Fn2 FilePath (Boolean -> a) Unit
+foreign import open_ :: Fn4 FilePath String (Nullable FileMode) (JSCallback FileDescriptor) Unit
+foreign import read_ :: Fn6 FileDescriptor Buffer BufferOffset BufferLength (Nullable FilePosition) (JSCallback ByteCount) Unit
+foreign import write_ :: Fn6 FileDescriptor Buffer BufferOffset BufferLength (Nullable FilePosition) (JSCallback ByteCount) Unit
+foreign import close_ :: Fn2 FileDescriptor (JSCallback Unit) Unit
```

Comment:
We can keep this as-is, but typically we'd name this

```purs
foreign import closeImpl :: ...
```

where `*Impl` denotes an FFI implementation

Reply:
Admittedly, this library already diverged from our typical best practices with this big `fs` record 😆